### DISCUSSION TITLE
GHES v4 availability?

### DISCUSSION THREAD

**Discussion by colinfish-toast** [actions/upload-artifact#603] [Q&A]
With the impending deprecation of v3, when can we expect v4 to be available on GHES?

**Comment by colinfish-toast** [comment]
[Announced in February, but pushed to Q4](https://github.com/github/roadmap/issues/930). [ankneis](https://github.com/ankneis), would you be able to provide a more concrete timeline for this?

**Comment by trevor-moore-dev** [comment]
My team and I would also love for this to be available on GHES, especially since large artifacts take so long to upload using v3, we've been resorting to caching instead

**Comment by trevor-moore-dev** [comment]
Is there any update on the timeline for GHE v4 availability?
